.evergreen/functions.yml

```diff
@@ -395,6 +395,25 @@ functions:
         file: signing_host_info.yml
 
   package:
+    - command: ec2.assume_role
+      display_name: Assume IAM role with permissions to pull Kondukto API token
+      params:
+        role_arn: ${kondukto_role_arn}
+    - command: shell.exec
+      display_name: Pull Kondukto API token from AWS Secrets Manager and write it to file
+      params:
+        silent: true
+        shell: bash
+        working_dir: src
+        include_expansions_in_env: [AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN]
+        script: |
+          # use AWS CLI to get the Kondukto API token from AWS Secrets Manager
+          kondukto_token=$(aws secretsmanager get-secret-value --secret-id "kondukto-token" --region "us-east-1" --query 'SecretString' --output text)
+          if [ $? -ne 0 ]; then
+              exit 1
+          fi
+          # set the KONDUKTO_TOKEN environment variable
+          echo "KONDUKTO_TOKEN=$kondukto_token" > ${workdir}/kondukto_credentials.env
```

Comment:
We tend to be very careful with placing files into `${workdir}` to avoid any risk of accidentally including the file in one of our published packages in the future – not necessarily requesting a change here, just letting you know why we previously wrote to `/tmp` and cleaned up the file instead of storing it here.